conventions/coding_style.md

```diff
@@ -4,7 +4,7 @@ This style is used in the standard library. You can use it in your own project t
 
 ## Naming
 
-__Type names__ are camelcased. For example:
+__Type names__ are StudlyCased. For example:
```

Comment:
I've never heard anyone call it this?

Reply:
Yeah, `CamelCased` please 👍

Reply:
I've typically found people refer to this as [PascalCase](https://techterms.com/definition/pascalcase).

```suggestion
__Type names__ are PascalCased. For example:
```